### ISSUE TITLE
Segmentation fault when calling twice the function get_symmetry

### ISSUE THREAD

**Issue by mesonepigreco** [singularitti/Spglib.jl#192]
**Seg fault**
When executing twice in a row the function get_symmetry, I get a segmentation fault

**To Reproduce**
1. Start Julia REPL
2. Run 
```julia
julia> using Spglib
julia> cell = [10.922687366041222 0.0 0.0; 5.46134368302061 9.459324736587035 0.0; 5.461343683020612 3.1531082455290136 8.919210198794858]
3×3 Matrix{Float64}:
 10.9227   0.0      0.0
  5.46134  9.45932  0.0
  5.46134  3.15311  8.91921

julia> pos =  [0.0 0.0 0.0; -8.326672684688674e-17 0.0 0.49999999999999994; 6.938893903907228e-17 0.4999999999999998 5.551115123125783e-17; 5.551115123125783e-17 0.4999999999999998 0.5; 0.49999999999999994 -1.3877787807814457e-17 -5.551115123125783e-17; 0.5000000000000002 0.0 0.4999999999999998; 0.49999999999999994 0.4999999999999996 0.0; 0.5 0.4999999999999997 0.4999999999999998]
8×3 Matrix{Float64}:
  0.0           0.0           0.0
 -8.32667e-17   0.0           0.5
  6.93889e-17   0.5           5.55112e-17
  5.55112e-17   0.5           0.5
  0.5          -1.38778e-17  -5.55112e-17
  0.5           0.0           0.5
  0.5           0.5           0.0
  0.5           0.5           0.5

julia> types = [1, 1, 1, 1, 1, 1, 1, 1]
8-element Vector{Int64}:
 1
 1
 1
 1
 1
 1
 1
 1

julia> cellspglib = Cell(cell', pos', types)
SpglibCell{Float64, Float64, Int64, Any}
 lattice:
   10.922687366041222  5.46134368302061  5.461343683020612
   0.0  9.459324736587035  3.1531082455290136
   0.0  0.0  8.919210198794858
 8 atomic positions:
   0.0  0.0  0.0
   -8.326672684688674e-17  0.0  0.49999999999999994
   6.938893903907228e-17  0.4999999999999998  5.551115123125783e-17
   5.551115123125783e-17  0.4999999999999998  0.5
   0.49999999999999994  -1.3877787807814457e-17  -5.551115123125783e-17
   0.5000000000000002  0.0  0.4999999999999998
   0.49999999999999994  0.4999999999999996  0.0
   0.5  0.4999999999999997  0.4999999999999998
 8 atoms:
   1  1  1  1  1  1  1  1


julia> R,T = Spglib.get_symmetry(cellspglib)
# ... 
julia> R,T = Spglib.get_symmetry(cellspglib)
Unreachable reached at 0x785068d560c3

[30456] signal (4.2): Illegal instruction
in expression starting at REPL[8]:1
== at /home/darth-vader/.julia/packages/SumTypes/aO6qd/src/sum_type.jl:294
unknown function (ip: 0x784ff778250f)
Allocations: 8597543 (Pool: 8588317; Big: 9226); GC: 14
Illegal instruction (core dumped)
```

**Versions (please complete the following information):**
 - OS: Ubuntu 22.04.04
 - Julia version: 1.10.4
 - Package version: 0.9.3

**Comment by singularitti** [comment]
Sorry, I could not reproduce on my macOS. Can you try with Julia v1.11.1?

**Comment by singularitti** [comment]
Also, could you please try version v0.9.4? This issue might be fixed after #183.

**Comment by singularitti** [comment]
Closed due to no response.
